src/Pages/LeaderBoard.js

```diff
@@ -58,12 +131,12 @@ function LeaderBoard() {
                                       </tr>
                                     </thead>
                                     <tbody id="protocols-tbody" className="divide-y divide-gray-200">
-                                      {
-                                        data
+                                      {  
+                                        allProtocols
                                         .filter( (item) => {
                                           if (searchTerm===""){
                                             return item
-                                          } else if (item.name.toLowerCase().includes(searchTerm.toLowerCase())){
+                                          } else if (item[0].toLowerCase().includes(searchTerm.toLowerCase())){
```

Comment:
I think we need to relook at the datastructure of "allProtocols", its really hard to see what these values should be when everything is a nested array. I reckon objects would be better.  Something like: 

```
[
{name: "maker", VAR: "20123", rawAssetData: [...]}, ...
]
```